kuksa_viss_client/__main__.py

```diff
@@ -11,16 +11,21 @@
 import argparse, json, sys
 from re import sub
 from typing import Dict, List
-import queue, time, os
+import queue, time, os, threading
 from pygments import highlight, lexers, formatters
 from cmd2 import Cmd, with_argparser, with_category, Cmd2ArgumentParser, CompletionItem
 from cmd2.utils import CompletionError, basic_complete
 import functools, subprocess
-DEFAULT_SERVER_ADDR = "127.0.0.1"
+DEFAULT_SERVER_ADDR = "172.17.0.1"
```

Comment:
Change back to 127.0.0.1

Reply:
Chaned with [31dc5a4](https://github.com/eclipse/kuksa.val/pull/291/commits/31dc5a4d0b7a0c1c4fe89a87f0b3426e7f65d9f6)